build/webpack.karma.config.js

```diff
@@ -1,19 +1,28 @@
 require('babel-polyfill');
 
+const { argv } = process;
```

Comment:
Looking for a better way to determine the original command to run  the tests

Reply:
We do this in webpack.config.js, would it work here? 
```
const isRelease = process.env.NODE_ENV === 'production';
const isDev = process.env.NODE_ENV === 'dev';
```

Reply:
That's a good catch to turn them off for prod, but we also want to turn them off if we're not running 'debug' in any environment. We never run debug during a production build so it should already be covered.

Reply:
ah good point.